code/__HELPERS/game.dm

```diff
@@ -519,9 +519,12 @@
 	if((character.mind.assigned_role == "Cyborg") || (character.mind.assigned_role == character.mind.special_role))
 		return
 
+	var/displayed_rank = rank//Tegu edit - Alt job titles
```

Comment:
Should make the comment here clearer, perhaps put it at the beginning, like for the edit end comment